static/css/main.css

```
@@ -11,15 +11,27 @@ label { display:inline-block; font-weight: 600; margin-bottom: 0.25rem; width:20
 }
 
 .field-container {
-    display:block;
     margin: 0 0 0.2rem 0;
     position: relative;
 }
 
-.field-container span {
+.hidden-container {
+    display:none;
+}
+
+
+.field-container__span {
     position: relative;
 }
 
+.group-title{
+    display:none;
+}
+
+.metadata-fields{
+    display:none;
+}
+
```

I think these should be renamed `.field-container--hidden`, `.group-title--hidden` and `.metadata-fields` to make it more descriptive and fit with BEM class naming. These will then need to be updated in the html. We can also make it much simpler buy setting them all at the same time like this:
```suggestion
.field-container--hidden, .metadata-fields-hidden, .group-title-hidden { 
    display:none;
}

.field-container__span {
    position: relative;
}
```

---

Added now 👍